ui/src/routes.ts

```diff
@@ -81,4 +82,11 @@ export const routes: Array<Route> = [
     enabled: true,
     component: ProfileContainer,
   },
+  {
+    key: 'songsView-route',
+    title: 'songsView',
+    path: '/songs/:id',
+    enabled: true,
+    component: SongsView,
```

Comment:
premissions will need to be added i believe

Reply:
yea think will resolve that when we merge tam's pr too, or else it will throw error rn lol